graph/db/addr.go

```diff
@@ -200,6 +227,24 @@ func DeserializeAddr(r io.Reader) (net.Addr, error) {
 			OnionService: onionService,
 			Port:         port,
 		}
+
+	case opaqueAddrs:
```

Comment:
do we need to also fix it for the migration where we have a similar method ?

Saw the AddressType Check in some `ReadElement` functions

Reply:
migrations run against any code we currently have persisted. Since up until this point we have not persisted node announcements with these unknown address types, old migrations should not require this. 

This is also why we do lots of codec copy/pasting for migrations - so that we can be sure that the migrations are using the serialisation code that existed at that point in time